src/python/platforms/fuchsia/util/device.py

```diff
@@ -12,8 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Utilities for managing Fuchsia devices."""
+from __future__ import absolute_import
+from builtins import object
+from builtins import range
 
-from host import Host
+from .host import Host
```

Comment:
can you move it after the system imports.

Reply:
Done